apps/meteor/server/methods/browseChannels.ts

```diff
@@ -289,6 +290,7 @@ const getUsers = async (
 
 			// Add the federated user to the results
 			results.unshift({
+				_id: user._id,
```

Comment:
Oof, the lack of this id was probably breaking a lot of stuff on the UI.

Reply:
![image](https://github.com/user-attachments/assets/fe59811a-ed99-4f56-a75c-d914c11085aa)